### DISCUSSION TITLE
Mobile Large file upload has issues

### DISCUSSION THREAD

**Discussion by makovez** [immich-app/immich#13303] [Q&A]
### The bug

It's been like 1/2 weeks that immich is not able to upload a file large 3-4gb. The issue is with mobile. 

Also it seems like when closing app it doesent keep uploading.

I have one plus 9 pro. And immich 1.116.2 on both server and app.

### The OS that Immich Server is running on

Debian 12

### Version of Immich Server

1.116.2

### Version of Immich Mobile App

1.116.2

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
.
```

### Your .env content

```Shell
.
```

### Reproduction steps

1.
2.
3.
...
.

**Comment by bo0tzz** [comment]
Are you using a reverse proxy, cloudflare tunnel, or anything of the sorts?

**Reply by makovez** [reply]
yes but i dont think it's a proxy or large file issues because when i keep immich mobile app on upload page it doesen't fail and keeps uploading. Just when i exit from app (without even closing) and re-open it starts again.

**Reply by makovez** [reply]
Actually keeping the mobile app on upload page (backup) it stops at 2.24/3.25GB

**Reply by bo0tzz** [reply]
It's almost definitely a proxy/large file issue ;)

Note that the foreground backup is separate and only runs when the app is open. If you close the app, it's expected to stop.

**Reply by makovez** [reply]
Regarding this yes, it probably is and i just updated my nginx config > Actually keeping the mobile app on upload page (backup) it stops at 2.24/3.25GB

But regarding the backup, shouldn't immich upload files on background when i take new videos/pictures ? The option is enabled in my app.  Also keep in mind that the upload stops even if I don't close the app from the background. If i just press home it still stops. 

<img src="https://github.com/user-attachments/assets/3178ead5-b81f-467e-9788-76369733ab1f" width="200"/>

**Reply by bo0tzz** [reply]
> shouldn't immich upload files on background when i take new videos/pictures ?

Yes, but that's separate from the backup process you're looking at in the app.
